src/docs/patterns/navigation.mdx

```diff
@@ -68,3 +86,31 @@ In this implementation, the navigation panel slides out from the side, partially
 The icon navigation bar (a.k.a. "[navigation rail](https://material.io/components/navigation-rail/#anatomy)" on Material.io) is a minimalist approach to the navigation panel. It features a single icon to represent each of the main areas of the application. This navigation panel is typically used on larger screens only, and is expandable to a full side navigation panel on hover or when clicking the menu icon.
 
 These may be used in conjunction with Bottom Tab navigation - on larger resolution displays, bottom tabs would become elements in the side navigation.
+
+## PX Blue Variations
+
+<ImageGrid images={[GenericExample1]} regularWidth />
+
+PX Blue provides the foundation for a wide range of navigation patterns. Ultimately the product will determine which pattern is best to use. The following sections provide PX Blue recommendations for common variation styles and standards on which variation to use in your product.
+
+### Nested Navigation Items
+
+PX Blue provides patterns to allow products to nest multiple layers of navigation options in the Navigation Menu. It is recommended not to exceed 2 levels of nested navigation. The graphic below illustrates the preferred style for nested navigation items.
```

Comment:
"PX Blue provides patterns to allow products to" => "You can"

Reply:
Generally, we spell out numbers less than ten, but I'm not sure how strictly we follow that.

Reply:
"The graphic below illustrates" => Give the image a caption